src/Solvers/fft_based_poisson_solver.jl

```diff
@@ -7,6 +10,26 @@ struct FFTBasedPoissonSolver{A, G, Λ, S, B, T}
       transforms :: T
 end
 
+transform_str(transform) = string(typeof(transform).name.wrapper, ", ")
+
+function transform_list_str(transform_list)
+    transform_strs = (transform_str(t) for t in transform_list)
+    list = string(transform_strs...)
+    list = list[1:end-2]
+    return list
+end
+
+Base.show(io::IO, solver::FFTBasedPoissonSolver{A, G}) where {A, G}=
+    print(io, "FFTBasedPoissonSolver{$A}: \n",
+          "├── grid: $(short_show(solver.grid))\n",
+          "├── storage: $(typeof(solver.storage))\n",
+          "├── buffer: $(typeof(solver.buffer))\n",
+          "└── transforms:\n",
+          "    ├── forward: ", transform_list_str(solver.transforms.forward), "\n",
+          "    └── backward: ", transform_list_str(solver.transforms.backward),
+         )
+          #"└── transforms: $(solver.transforms)")
```

Comment:
Do we need the commented out code?

Reply:
Good catch, no. We list the forward and backward transforms separately instead.